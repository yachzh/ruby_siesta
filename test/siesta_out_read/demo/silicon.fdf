SystemLabel   silicon
NumberOfAtoms   2
NumberOfSpecies   1
LatticeConstant   1.0 Ang
%block LatticeVectors
     0.00000000      2.71500000      2.71500000
     2.71500000      0.00000000      2.71500000
     2.71500000      2.71500000      0.00000000
%endblock LatticeVectors

AtomicCoordinatesFormat   Ang
%block ChemicalSpeciesLabel
  1   14   Si
%endblock ChemicalSpeciesLabel

%block AtomicCoordinatesAndAtomicSpecies
     0.00000000      0.00000000      0.00000000   1
     1.35750000      1.35750000      1.35750000   1
%endblock AtomicCoordinatesAndAtomicSpecies

PAO.BasisSize   DZP
PAO.EnergyShift   50 meV
Mesh.Cutoff   200 Ry
WriteMullikenPop   1
MullikenInSCF   false
WriteCoorXmol   true
SCF.Mixer.Weight   0.01
SCF.Mixer.History   5
SCF.Mixer.Kick   0
SCF.DM.Tolerance   0.0001
SCF.H.Converge   true
SCF.H.Tolerance   1.0 meV
MaxSCFIterations   2000
DM.UseSaveDM   true
OccupationFunction   MP
OccupationMPOrder   4
ElectronicTemperature   1000 K
NetCharge   0
Slab.DipoleCorrection   false
XC.Functional   GGA
XC.Authors   PBE
Spin   non-polarized
Spin.Fix   false
Spin.Total   0

%block kgrid_Monkhorst_Pack
  9   0   0   0.0
  0   9   0   0.0
  0   0   9   0.0
%endblock kgrid_Monkhorst_Pack
