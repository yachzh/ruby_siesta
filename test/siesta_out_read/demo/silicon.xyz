    2

Si      0.000000    0.000000    0.000000
Si      1.357500    1.357500    1.357500
