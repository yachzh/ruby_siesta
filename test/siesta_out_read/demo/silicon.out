Siesta Version  : 4.1.5
Architecture    : arm64
Compiler version: GNU Fortran (Homebrew GCC 12.2.0) 12.2.0
Compiler flags  : mpif90 -O2 -fPIC -ftree-vectorize -fallow-argument-mismatch
PP flags        : -DCDF -DNCDF -DNCDF_4 -DMPI -DFC_HAVE_ABORT
Libraries       : libncdf.a libfdict.a -L/opt/homebrew/lib -lnetcdff -lnetcdf -lhdf5_hl -lhdf5  -lz -lscalapack -L/opt/homebrew/opt/openblas/lib -lopenblas
PARALLEL version
NetCDF support
NetCDF-4 support

* Running on 4 nodes in parallel
>> Start of run:  26-JUL-2024  11:54:55

                           ***********************       
                           *  WELCOME TO SIESTA  *       
                           ***********************       

reinit: Reading from silicon.fdf

reinit: -----------------------------------------------------------------------
reinit: System Name: 
reinit: -----------------------------------------------------------------------
reinit: System Label: silicon
reinit: -----------------------------------------------------------------------

initatom: Reading input for the pseudopotentials and atomic orbitals ----------
Species number:   1 Atomic number:   14 Label: Si

Ground state valence configuration:   3s02  3p02
Reading pseudopotential information in formatted form from Si.psf

Valence configuration for pseudopotential generation:
3s( 2.00) rc: 1.77
3p( 2.00) rc: 1.96
3d( 0.00) rc: 2.11
4f( 0.00) rc: 2.11
For Si, standard SIESTA heuristics set lmxkb to 3
 (one more than the basis l, including polarization orbitals).
Use PS.lmax or PS.KBprojectors blocks to override.

<basis_specs>
===============================================================================
Si                   Z=  14    Mass=  28.090        Charge= 0.17977+309
Lmxo=1 Lmxkb= 3    BasisType=split      Semic=F
L=0  Nsemic=0  Cnfigmx=3
          i=1  nzeta=2  polorb=0  (3s)
            splnorm:   0.15000    
               vcte:    0.0000    
               rinn:    0.0000    
               qcoe:    0.0000    
               qyuk:    0.0000    
               qwid:   0.10000E-01
                rcs:    0.0000      0.0000    
            lambdas:    1.0000      1.0000    
L=1  Nsemic=0  Cnfigmx=3
          i=1  nzeta=2  polorb=1  (3p)
            splnorm:   0.15000    
               vcte:    0.0000    
               rinn:    0.0000    
               qcoe:    0.0000    
               qyuk:    0.0000    
               qwid:   0.10000E-01
                rcs:    0.0000      0.0000    
            lambdas:    1.0000      1.0000    
-------------------------------------------------------------------------------
L=0  Nkbl=1  erefs: 0.17977+309
L=1  Nkbl=1  erefs: 0.17977+309
L=2  Nkbl=1  erefs: 0.17977+309
L=3  Nkbl=1  erefs: 0.17977+309
===============================================================================
</basis_specs>

atom: Called for Si                    (Z =  14)

read_vps: Pseudopotential generation method:
read_vps: ATM 3.2.2 Troullier-Martins                       
Valence charge for ps generation:    4.00000

xc_check: Exchange-correlation functional:
xc_check: GGA Perdew, Burke & Ernzerhof 1996
V l=0 = -2*Zval/r beyond r=  1.7304
V l=1 = -2*Zval/r beyond r=  1.9124
V l=2 = -2*Zval/r beyond r=  2.0613
V l=3 = -2*Zval/r beyond r=  2.0613
All V_l potentials equal beyond r=  2.0613
This should be close to max(r_c) in ps generation
All pots = -2*Zval/r beyond r=  2.0613

VLOCAL1: 99.0% of the norm of Vloc inside     10.216 Ry
VLOCAL1: 99.9% of the norm of Vloc inside     23.282 Ry
atom: Maximum radius for 4*pi*r*r*local-pseudopot. charge    2.45560
atom: Maximum radius for r*vlocal+2*Zval:    2.14012
GHOST: No ghost state for L =  0
GHOST: No ghost state for L =  1
GHOST: No ghost state for L =  2
GHOST: No ghost state for L =  3

KBgen: Kleinman-Bylander projectors: 
   l= 0   rc=  2.278153   el= -0.795700   Ekb=  4.691105   kbcos=  0.294879
   l= 1   rc=  2.306811   el= -0.300424   Ekb=  1.499986   kbcos=  0.307489
   l= 2   rc=  2.335829   el=  0.001958   Ekb= -2.261682   kbcos= -0.053994
   l= 3   rc=  2.425092   el=  0.003046   Ekb= -0.701404   kbcos= -0.005423

KBgen: Total number of Kleinman-Bylander projectors:   16
atom: -------------------------------------------------------------------------

atom: SANKEY-TYPE ORBITALS:
atom: Selected multiple-zeta basis: split     

SPLIT: Orbitals with angular momentum L= 0

SPLIT: Basis orbitals for state 3s

SPLIT: PAO cut-off radius determined from an
SPLIT: energy shift=  0.003675 Ry

   izeta = 1
                 lambda =    1.000000
                     rc =    6.116033
                 energy =   -0.792669
                kinetic =    0.531445
    potential(screened) =   -1.324114
       potential(ionic) =   -3.756103

   izeta = 2
                 rmatch =    4.530823
              splitnorm =    0.150000
                 energy =   -0.698051
                kinetic =    0.851639
    potential(screened) =   -1.549691
       potential(ionic) =   -4.098970

SPLIT: Orbitals with angular momentum L= 1

SPLIT: Basis orbitals for state 3p

SPLIT: PAO cut-off radius determined from an
SPLIT: energy shift=  0.003675 Ry

   izeta = 1
                 lambda =    1.000000
                     rc =    7.853192
                 energy =   -0.296941
                kinetic =    0.798182
    potential(screened) =   -1.095123
       potential(ionic) =   -3.295985

   izeta = 2
                 rmatch =    5.330310
              splitnorm =    0.150000
                 energy =   -0.227975
                kinetic =    1.148906
    potential(screened) =   -1.376881
       potential(ionic) =   -3.769803

POLgen: Perturbative polarization orbital with L=  2

POLgen: Polarization orbital for state 3p

   izeta = 1
                     rc =    7.853192
                 energy =    0.340575
                kinetic =    1.096546
    potential(screened) =   -0.755971
       potential(ionic) =   -2.720331
atom: Total number of Sankey-type orbitals: 13

atm_pop: Valence configuration (for local Pseudopot. screening):
 3s( 2.00)                                                            
 3p( 2.00)                                                            
Vna: chval, zval:    4.00000   4.00000

Vna:  Cut-off radius for the neutral-atom potential:   7.853192

atom: _________________________________________________________________________

prinput: Basis input ----------------------------------------------------------

PAO.BasisType split     

%block ChemicalSpeciesLabel
    1   14 Si                      # Species index, atomic number, species label
%endblock ChemicalSpeciesLabel

%block PAO.Basis                 # Define Basis set
Si                    2                    # Species label, number of l-shells
 n=3   0   2                         # n, l, Nzeta 
   6.116      4.531   
   1.000      1.000   
 n=3   1   2 P   1                   # n, l, Nzeta, Polarization, NzetaPol
   7.853      5.330   
   1.000      1.000   
%endblock PAO.Basis

prinput: ----------------------------------------------------------------------

Dumping basis to NetCDF file Si.ion.nc
coor:   Atomic-coordinates input format  =     Cartesian coordinates
coor:                                          (in Angstroms)

siesta: Atomic coordinates (Bohr) and species
siesta:      0.00000   0.00000   0.00000  1        1
siesta:      2.56530   2.56530   2.56530  1        2

siesta: System type = bulk      

initatomlists: Number of atoms, orbitals, and projectors:      2    26    32

coxmol: Writing XMOL coordinates into file silicon.xyz                                                         

siesta: ******************** Simulation parameters ****************************
siesta:
siesta: The following are some of the parameters of the simulation.
siesta: A complete list of the parameters used, including default values,
siesta: can be found in file out.fdf
siesta:
redata: Spin configuration                          = none
redata: Number of spin components                   = 1
redata: Time-Reversal Symmetry                      = T
redata: Spin spiral                                 = F
redata: Long output                                 =   F
redata: Number of Atomic Species                    =        1
redata: Charge density info will appear in .RHO file
redata: Write Mulliken Pop.                         = Atomic and Orbital charges
redata: Matel table size (NRTAB)                    =     1024
redata: Mesh Cutoff                                 =   200.0000 Ry
redata: Net charge of the system                    =     0.0000 |e|
redata: Min. number of SCF Iter                     =        0
redata: Max. number of SCF Iter                     =     2000
redata: SCF convergence failure will abort job
redata: SCF mix quantity                            = Hamiltonian
redata: Mix DM or H after convergence               =   F
redata: Recompute H after scf cycle                 =   F
redata: Mix DM in first SCF step                    =   T
redata: Write Pulay info on disk                    =   F
redata: New DM Occupancy tolerance                  = 0.000000000001
redata: No kicks to SCF
redata: DM Mixing Weight for Kicks                  =     0.5000
redata: Require Harris convergence for SCF          =   F
redata: Harris energy tolerance for SCF             =     0.000100 eV
redata: Require DM convergence for SCF              =   T
redata: DM tolerance for SCF                        =     0.000100
redata: Require EDM convergence for SCF             =   F
redata: EDM tolerance for SCF                       =     0.001000 eV
redata: Require H convergence for SCF               =   T
redata: Hamiltonian tolerance for SCF               =     0.001000 eV
redata: Require (free) Energy convergence for SCF   =   F
redata: (free) Energy tolerance for SCF             =     0.000100 eV
redata: Using Saved Data (generic)                  =   F
redata: Use continuation files for DM               =   T
redata: Neglect nonoverlap interactions             =   F
redata: Method of Calculation                       = Diagonalization
redata: Electronic Temperature                      =   999.9927 K
redata: Fix the spin of the system                  =   F
redata: Dynamics option                             = Single-point calculation
mix.SCF: Pulay mixing                            = Pulay
mix.SCF:    Variant                              = stable
mix.SCF:    History steps                        = 5
mix.SCF:    Linear mixing weight                 =     0.010000
mix.SCF:    Mixing weight                        =     0.010000
mix.SCF:    SVD condition                        = 0.1000E-07
redata: Save all siesta data in one NC              =   F
redata: ***********************************************************************

%block SCF.Mixers
  Pulay
%endblock SCF.Mixers

%block SCF.Mixer.Pulay
  # Mixing method
  method pulay
  variant stable

  # Mixing options
  weight 0.0100
  weight.linear 0.0100
  history 5
%endblock SCF.Mixer.Pulay

DM_history_depth set to one: no extrapolation allowed by default for geometry relaxation
Size of DM history Fstack: 1
Total number of electrons:     8.000000
Total ionic charge:     8.000000

* ProcessorY, Blocksize:    2   7


* Orbital distribution balance (max,min):     7     5

 Kpoints in:          405 . Kpoints trimmed:          365

siesta: k-grid: Number of k-points =   365
siesta: k-grid: Cutoff (effective) =    17.278 Ang
siesta: k-grid: Supercell and displacements
siesta: k-grid:    9   0   0      0.000
siesta: k-grid:    0   9   0      0.000
siesta: k-grid:    0   0   9      0.000

diag: Algorithm                                     = D&C
diag: Parallel over k                               =   F
diag: Use parallel 2D distribution                  =   T
diag: Parallel block-size                           = 7
diag: Parallel distribution                         =     2 x     2
diag: Used triangular part                          = Lower
diag: Absolute tolerance                            =  0.100E-15
diag: Orthogonalization factor                      =  0.100E-05
diag: Memory factor                                 =  1.0000

superc: Internal auxiliary supercell:     5 x     5 x     5  =     125
superc: Number of atoms, orbitals, and projectors:    250   3250   4000


ts: **************************************************************
ts: Save H and S matrices                           =    F
ts: Save DM and EDM matrices                        =    F
ts: Only save the overlap matrix S                  =    F
ts: **************************************************************

************************ Begin: TS CHECKS AND WARNINGS ************************
************************ End: TS CHECKS AND WARNINGS **************************


                     ====================================
                        Single-point calculation
                     ====================================

superc: Internal auxiliary supercell:     5 x     5 x     5  =     125
superc: Number of atoms, orbitals, and projectors:    250   3250   4000

outcell: Unit cell vectors (Ang):
        0.000000    2.715000    2.715000
        2.715000    0.000000    2.715000
        2.715000    2.715000    0.000000

outcell: Cell vector modules (Ang)   :    3.839590    3.839590    3.839590
outcell: Cell angles (23,13,12) (deg):     60.0000     60.0000     60.0000
outcell: Cell volume (Ang**3)        :     40.0258
<dSpData1D:S at geom step 0
  <sparsity:sparsity for geom step 0
    nrows_g=26 nrows=7 sparsity=17.4216 nnzs=11777, refcount: 7>
  <dData1D:(new from dSpData1D) n=11777, refcount: 1>
refcount: 1>
new_DM -- step:     1
Initializing Density Matrix...

Attempting to read DM from file... Failed...
DM filled with atomic data:
<dSpData2D:DM initialized from atoms
  <sparsity:sparsity for geom step 0
    nrows_g=26 nrows=7 sparsity=17.4216 nnzs=11777, refcount: 8>
  <dData2D:DM n=11777 m=1, refcount: 1>
refcount: 1>
No. of atoms with KB's overlaping orbs in proc 0. Max # of overlaps:      35     365
New grid distribution:   1
           1       1:   15    1:    8    1:    8
           2       1:   15    1:    8    9:   15
           3       1:   15    9:   15    1:    8
           4       1:   15    9:   15    9:   15

InitMesh: MESH =    30 x    30 x    30 =       27000
InitMesh: (bp) =    15 x    15 x    15 =        3375
InitMesh: Mesh cutoff (required, used) =   200.000   253.085 Ry
ExtMesh (bp) on 0 =    95 x    88 x    88 =      735680
New grid distribution:   2
           1       1:   15    1:    7    1:    7
           2       1:   15    1:    7    8:   15
           3       1:   15    8:   15    1:    7
           4       1:   15    8:   15    8:   15
New grid distribution:   3
           1       1:   15    1:    8    1:    8
           2       1:   15    1:    8    9:   15
           3       1:   15    9:   15    1:    8
           4       1:   15    9:   15    9:   15
Setting up quadratic distribution...
ExtMesh (bp) on 0 =    95 x    87 x    87 =      719055
PhiOnMesh: Number of (b)points on node 0 =                  735
PhiOnMesh: nlist on node 0 =               116359

stepf: Methfessel-Paxton step function
       Using Hermite-Gauss polynomials of order  4

siesta: Program's energy decomposition (eV):
siesta: Ebs     =       -84.070430
siesta: Eions   =       377.610450
siesta: Ena     =       116.808398
siesta: Ekin    =        82.133700
siesta: Enl     =        29.022837
siesta: Eso     =         0.000000
siesta: Edftu   =         0.000000
siesta: DEna    =         0.521336
siesta: DUscf   =         0.457186
siesta: DUext   =         0.000000
siesta: Exc     =       -65.498469
siesta: eta*DQ  =         0.000000
siesta: Emadel  =         0.000000
siesta: Emeta   =         0.000000
siesta: Emolmec =         0.000000
siesta: Ekinion =         0.000000
siesta: Eharris =      -210.538842
siesta: Etot    =      -214.165462
siesta: FreeEng =      -214.165462

        iscf     Eharris(eV)        E_KS(eV)     FreeEng(eV)     dDmax    Ef(eV) dHmax(eV)
   scf:    1     -210.538842     -214.165462     -214.165462  1.744622 -5.386614  0.473947
timer: Routine,Calls,Time,% = IterSCF        1       1.477  75.29
   scf:    2     -214.166060     -214.165761     -214.165761  0.001027 -5.383414  0.468346
   scf:    3     -214.188428     -214.178132     -214.178117  0.080560 -5.364896  0.002009
   scf:    4     -214.178130     -214.178131     -214.178116  0.000138 -5.364215  0.002285
   scf:    5     -214.178140     -214.178136     -214.178121  0.000818 -5.365160  0.000135
mix: Pulay -- inversion failed, > SVD [rank/size] 2 / 4
   scf:    6     -214.178136     -214.178136     -214.178121  0.000017 -5.365181  0.000107

SCF Convergence by DM+H criterion
max |DM_out - DM_in|         :     0.0000173730
max |H_out - H_in|      (eV) :     0.0001071183
SCF cycle converged after 6 iterations

Using DM_out to compute the final energy and forces
No. of atoms with KB's overlaping orbs in proc 0. Max # of overlaps:      35     365

siesta: E_KS(eV) =             -214.1781

siesta: E_KS - E_eggbox =      -214.1781

siesta: Atomic forces (eV/Ang):
----------------------------------------
   Tot    0.000001    0.000001    0.000001
----------------------------------------
   Max    0.000001
   Res    0.000001    sqrt( Sum f_i^2 / 3N )
----------------------------------------
   Max    0.000001    constrained

Stress tensor Voigt[x,y,z,yz,xz,xy] (kbar):      -45.18      -45.18      -45.18       -0.00       -0.00       -0.00
(Free)E + p*V (eV/cell)     -213.0494
Target enthalpy (eV/cell)     -214.1781

mulliken: Atomic and Orbital Populations:

Species: Si                  
Atom  Qatom  Qorb
               3s      3s      3py     3pz     3px     3py     3pz     3px     
               3Pdxy   3Pdyz   3Pdz2   3Pdxz   3Pdx2-y2
   1  3.999   0.608   0.593   0.203   0.203   0.203   0.555   0.555   0.555
              0.125   0.125   0.075   0.125   0.075
   2  4.001   0.606   0.594   0.208   0.208   0.208   0.551   0.551   0.551
              0.125   0.125   0.075   0.125   0.075

mulliken: Qtot =        8.000

coxmol: Writing XMOL coordinates into file silicon.xyz                                                         

siesta: Program's energy decomposition (eV):
siesta: Ebs     =       -84.544338
siesta: Eions   =       377.610450
siesta: Ena     =       116.808398
siesta: Ekin    =        81.454030
siesta: Enl     =        28.767620
siesta: Eso     =         0.000000
siesta: Edftu   =         0.000000
siesta: DEna    =         1.293392
siesta: DUscf   =         0.391579
siesta: DUext   =         0.000000
siesta: Exc     =       -65.282706
siesta: eta*DQ  =         0.000000
siesta: Emadel  =         0.000000
siesta: Emeta   =         0.000000
siesta: Emolmec =         0.000000
siesta: Ekinion =         0.000000
siesta: Eharris =      -214.178136
siesta: Etot    =      -214.178136
siesta: FreeEng =      -214.178121

siesta: Final energy (eV):
siesta:  Band Struct. =     -84.544338
siesta:       Kinetic =      81.454030
siesta:       Hartree =      14.410111
siesta:       Edftu   =       0.000000
siesta:       Eso     =       0.000000
siesta:    Ext. field =       0.000000
siesta:   Exch.-corr. =     -65.282706
siesta:  Ion-electron =    -109.891536
siesta:       Ion-ion =    -134.868035
siesta:       Ekinion =       0.000000
siesta:         Total =    -214.178136
siesta:         Fermi =      -5.365181

siesta: Stress tensor (static) (eV/Ang**3):
siesta:    -0.028200   -0.000002   -0.000002
siesta:    -0.000002   -0.028200   -0.000002
siesta:    -0.000002   -0.000002   -0.028200

siesta: Cell volume =         40.025752 Ang**3

siesta: Pressure (static):
siesta:                Solid            Molecule  Units
siesta:           0.00030714          0.00030714  Ry/Bohr**3
siesta:           0.02820018          0.02820014  eV/Ang**3
siesta:          45.18215490         45.18209918  kBar
(Free)E+ p_basis*V_orbitals  =        -211.943689
(Free)Eharris+ p_basis*V_orbitals  =        -211.943689

cite: Please see "silicon.bib" for an exhaustive BiBTeX file.
cite: Please clearly indicate Siesta version in published work: 4.1.5
cite: This calculation has made use of the following articles
cite: which are encouraged to be cited in a published work.
        Primary SIESTA paper
          DOI: www.doi.org/10.1088/0953-8984/14/11/302

>> End of run:  26-JUL-2024  11:55:04
Job completed
